src/sw/controller.js

```diff
@@ -24,10 +24,17 @@ export class Controller {
 
     constructor () {
         this.clientId = getRetrievalClientId()
+        this.clientKey = getClientKey()
         this.saturn = new Saturn({
-            clientKey: 'abc123',
-            storage: indexedDbStorage()
+            clientKey: this.clientKey,
+            storage: indexedDbStorage(),
+            authURL: 'https://fz3dyeyxmebszwhuiky7vggmsu0rlkoy.lambda-url.us-west-2.on.aws',
+            orchURL: 'https://orchestrator.strn.pl/nodes',
+            cdnURL: 'https://l1s.saturn.ms',
+            originURL: 'djib.io',
```

Comment:
why's this set?

Reply:
I really only meant the `originURL` param, but yeah lets just stick to js-client defaults until we need to override.